tests/storage/test_id_generators.py

```diff
@@ -59,7 +58,7 @@ def _create(conn):
                 writers=writers,
             )
 
-        return self.get_success(self.db_pool.runWithConnection(_create))
+        return self.get_success_or_raise(self.db_pool.runWithConnection(_create))
```

Comment:
Wouldn't `get_success` fail anyway? Why it his change necessary?

Reply:
`get_success` fails the test if the deferred fails, which I find a bit unintuitive.

Reply:
Ahhh, right. I wonder if there are places we do this wrong. 😢

Reply:
Usually when waiting on a deferred you know whether you're expecting this to succeed (`get_success`) or fail (`get_failure` and co), so I think is a little bit of an edge case.